ui-new/html/hard-limit-reached.html

```diff
@@ -0,0 +1,15 @@
+<section class="viewport hard_limit_reached_state cash-in-color" data-tr-section="hard-limit-reached">
```

Comment:
I think this screen should work for both cash-in and cash-out, checkout `setDirection` on app.js on how to do this, the cash-in-color should be removed from this. Also make sure to add it to test-app.js so it doesn't start as a blank page while testing.